frontend/src/views/ShootList.vue

```diff
@@ -222,6 +220,22 @@ export default {
       this.clearSelectedShootTimerID = setTimeout(() => {
         this.setSelectedShootInternal(null)
       }, 500)
+    },
+    searchShoots (value, search, item) {
+      const storeGetters = {
+        ticketsLabels: this.ticketsLabels,
+        shootCustomFields: this.shootCustomFields,
+        shootCustomFieldList: this.shootCustomFieldList
+      }
+      return searchShoots(storeGetters, value, search, item)
+    },
+    sortShoots (items, sortByArr, sortDescArr) {
+      const storeGetters = {
+        ticketsLabels: this.ticketsLabels,
+        shootCustomFields: this.shootCustomFields,
+        latestUpdatedTicketByNameAndNamespace: this.latestUpdatedTicketByNameAndNamespace
+      }
+      return sortShoots(storeGetters, items, sortByArr, sortDescArr)
```

Comment:
I would implement these two functions:

Either as getters in the shoots store module
```js
filterItems (state, getters, rootState, rootGetters) {
  return (value, search, item) => {
    ...
  }
},
sortItems (state, getters, rootState, rootGetters) {
  return (value, search, item) => {
    ...
  }
},
```

or as component mixin `ShootDataTable` with access to the mixins.